### PR TITLE
Adapt kselftests module to use packages only

### PR DESCRIPTION
**Pull request by acerv**
This patch removes all sources build dependences and it adds packages support for multiple kselftests testing suites.

- Related ticket: https://progress.opensuse.org/issues/131180
- Verification run: https://openqa.suse.de/tests/13706574 https://openqa.suse.de/tests/13706573

### CODE REVIEW

**Review by czerw — APPROVED** [review]
LGTM , it looks more clean now, thank you.

**Review by pevik — APPROVED** [review]
LGTM, although I would keep also compilation as fallback. You never know when you will want to validate against mainline tarball.

File: lib/LTP/utils.pm [hunk 1]
```diff
@@ -432,4 +421,21 @@ sub install_from_repo {
     }
 }
 
+sub prepare_whitelist_environment {
```
Comment (mdoucha) [inline lib/LTP/utils.pm:424]: Please add this to `@EXPORT` so you don't need to prefix it with `LTP::utils::`.

**Review by mdoucha — APPROVED** [review]
Other than the function prefix inconvenience issue, LGTM.

**Pull request merged**